### ISSUE TITLE
Active button not changing color for "Filter Group Images by Topic"

### ISSUE THREAD

**Issue by erinchambers** [CDRH/bandhistory_rails#57]
Identified by Will during an a11y PR, then confirmed by Erin Kay. It's a bug on the production site. From Will's review:

> I tested /multimedia/images. When I click the buttons under Filter Group By Titles, they do not highlight. The search results change, but there is no indication of what has been filtered, by either a filter box, or changing the color of the button.

This page has the nav in question; the "Field []" button should have a red background to indicate that it's the active page, but it doesn't.
- Dev: https://cdrhrailsdev.unl.edu/bandhistory/multimedia/images/field
- Prod: https://bandhistory.unl.edu/multimedia/images/field

The bug is probably stemming from the `topic_selected?(facet)` code on this line: https://github.com/CDRH/bandhistory_rails/blob/174ca897fda2c4035ac555e1e7ab974ffc692698/app/views/items/images/_image_nav.html.erb#L11

There's a similar nav on https://cdrhrailsdev.unl.edu/bandhistory/multimedia/footage, but where .../multimedia/images uses `topic_selected(facet)`, .../multimedia/footage uses `footage_path({ "f" => ["keywords|#{keyword}"] })`. I couldn't figure out how to translate that to the image partial.

**Comment by erinchambers** [comment]
Marking this as a11y-related, since it involves a state change to indicate the active page.
